### PR TITLE
Update steps for WordPress setup

### PR DESCRIPTION
**Pull request by mamjad00**
* Previously we were using page templates for relevant pages and now we have removed some of page templates
* Add some extra points like setting the branding logo, favicon, footer section and menus
* Add link for ACF fields and Elementor `.json` template files
* Add link for setup WordPress multisite

### CODE REVIEW

**Comment by etechdevai** [comment]
> _No description provided._

@mamjad00 add PR description.

File: README.rst [hunk 1]
```diff
@@ -548,51 +548,37 @@ and then run the provision again.
...
-6. Click publish.
-
-To add **Course Additional Fields** follow the steps below:
-
-1. Log into WordPress admin panel.
-2. Go to "Custom Fields" -> "Tools" page.
-3. In "Import Field Groups", select and import ``<devstack>/edly-wp-theme/config-files/acf-export-english.json`` file.
+To setup **Wordpress** you need to login with **Super Admin** user and then follow these steps:
+
+- Add **Site Logo** and **Favicon** from `Appearance > Customize > Site Identity`
```
Comment (zubair-arbi) [inline README.rst]: @mamjad00 used numbered bullets
```
1. Step #1 
2. Step #2
```

**Review by zubair-arbi — APPROVED** [review]
@mamjad00 just one suggestion otherwise changes looks good 👍 
@iam-mhaseeb @asadmanzoor93 can review these changes as well since they have recently did WP setup.

**Review by etechdevai — APPROVED** [review]
👍

**Review by burhandodhy — APPROVED** [review]
👍

**Review by iam-mhaseeb — APPROVED** [review]
@mamjad00  Can we add a multisite setup link here as well?
https://edlyio.atlassian.net/wiki/spaces/PI/pages/645136408/WordPress+Setup+Guide

File: README.rst (hunk 1 above)
Comment (mamjad00) [reply README.rst]: Done

**Comment by mamjad00** [comment]
> > _No description provided._
> 
> @mamjad00 add PR description.

Done

**Pull request merged**